libexec/klab-compress

```diff
@@ -31,15 +31,14 @@ if [ ! -d "$KLAB_OUT/log" ]; then
     mkdir -p "$KLAB_OUT/log"
 fi
 
-zip -r "$KLAB_OUT/log/$spec_hash.zip" config.json
-
 cd $KLAB_OUT
 find . \
-     \( -wholename "*$spec_hash*" -or -wholename "*$spec_name*" \) \
-     -and ! -wholename "*/log/*.zip" \
-     -exec zip "./log/$spec_hash.zip" {} +
-
-zip -r "log/$spec_hash.zip" \
-    "prelude.smt2" \
-    "rules.k" \
-    "bin_runtime.k"
+  \( -name "*$spec_hash*" \
+  -o -name "*$spec_name*" \
+  -o -name config.tmp.json \
```

Comment:
I changed this from `config.json` to `config.tmp.json`, because the former is not in `$KLAB_OUT` while the latter is, and this made the code cleaner (since I only have to search in one dir).

AFAICT the former is a subset of the latter, so everything should be fine.

@mhhf true story?